web_tool_stress_test.R

```diff
@@ -221,7 +224,10 @@ scenario_data <- scenario_data %>%
 
 # %>% filter(year %in% c(start_year,2020, 2021, 2022, 2023, 2024, 2025, 2030, 2035, 2040))
 
-df_price <- readr::read_csv(file.path(data_location, glue::glue("prices_data_{price_data_version}.csv")), col_types = cols()) %>%
+df_price <- read_price_data(
+    path = file.path(data_location, paste0("prices_data_", price_data_version, ".csv")),
+    version = "old"
+  ) %>%
   filter(year >= start_year) %>%
```

Comment:
out of interest, where does start_year come from here? I did not find an assignment and assume it is somehow made available in he env when calling the script?

Reply:
project configuration